engine/modelc/src/java/com/dynamo/bob/pipeline/ModelImporter.java

```diff
@@ -197,28 +200,27 @@ public static class Animation {
         public float            duration;
     }
 
+    public static class Buffer {
+        public String           uri;
+        public byte[]           buffer;
+    }
```

Comment:
The new buffers exposed to the "client".
Mainly used to check if the resources _were_ loaded, and which resources to link with the main model resource.